packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -108,12 +108,12 @@ class TreeTable<T extends TreeNode<T>> extends StatefulWidget {
     Key key,
     @required this.columns,
     @required this.treeColumn,
-    @required this.data,
+    @required this.dataRoots,
```

Comment:
Update the dartdoc to explain what is happening here (and update the references to the [data] variable name.

I believe the dart analyzer rename refactoring can do this (in the cases where it doesn't time out).